src/wavesurfer.js

```diff
@@ -232,6 +233,18 @@ const WaveSurfer = util.extend({}, util.observer, { util }, {
         this.drawer = Object.create(this.Drawer);
         this.drawer.init(this.container, this.params);
         this.fireEvent('drawer-created', this.drawer);
+        let prevWidth = 0;
+        this._onResize = () => {
+            if (prevWidth != this.drawer.wrapper.clientWidth) {
+                prevWidth = this.drawer.wrapper.clientWidth;
+                this.empty();
+                this.drawBuffer();
+            }
+        };
+
+        if (this.params.responsive) {
+            window.addEventListener('resize', this._onResize, true);
+        }
```

Comment:
@thijstriemstra yea, maybe we can add a `orientationchange` check and event for mobile devices? On mobile you can zoom and change the orientation vertical/horizontal.